src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -911,6 +911,120 @@ public virtual CreatedAtRouteResult CreatedAtRoute(string routeName, object rout
+        public virtual AcceptedAtActionResult AcceptedAtAction(string actionName, object routeValues, object value)
+        {
+            return AcceptedAtAction(actionName, controllerName: null, routeValues: routeValues, value: value);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtActionResult"/> object that produces an Accepted (202) response.
+        /// </summary>
+        /// <param name="actionName">The name of the action to use for generating the URL.</param>
+        /// <param name="controllerName">The name of the controller to use for generating the URL.</param>
+        /// <param name="routeValues">The route data to use for generating the URL.</param>
+        /// <param name="value">The content value to format in the entity body.</param>
+        /// <returns>The created <see cref="AcceptedAtActionResult"/> for the response.</returns>
+        [NonAction]
+        public virtual AcceptedAtActionResult AcceptedAtAction(
+            string actionName,
+            string controllerName,
+            object routeValues,
+            object value)
+        {
+            return new AcceptedAtActionResult(actionName, controllerName, routeValues, value);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an Accepted (202) response.
+        /// </summary>
+        /// <param name="routeName">The name of the route to use for generating the URL.</param>
+        /// <param name="value">The content value to format in the entity body.</param>
+        /// <returns>The created <see cref="AcceptedAtRouteResult"/> for the response.</returns>
+        [NonAction]
+        public virtual AcceptedAtRouteResult AcceptedAtRoute(string routeName, object value)
+        {
+            return AcceptedAtRoute(routeName, routeValues: null, value: value);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an Accepted (202) response.
+        /// </summary>
+        /// <param name="routeValues">The route data to use for generating the URL.</param>
+        /// <param name="value">The content value to format in the entity body.</param>
+        /// <returns>The created <see cref="AcceptedAtRouteResult"/> for the response.</returns>
+        [NonAction]
+        public virtual AcceptedAtRouteResult AcceptedAtRoute(object routeValues, object value)
+        {
+            return AcceptedAtRoute(routeName: null, routeValues: routeValues, value: value);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedAtRouteResult"/> object that produces an Accepted (202) response.
+        /// </summary>
+        /// <param name="routeName">The name of the route to use for generating the URL.</param>
+        /// <param name="routeValues">The route data to use for generating the URL.</param>
+        /// <param name="value">The content value to format in the entity body.</param>
```

Comment:
No, we don't use default values in public APIs for various reasons. I wouldn't make any changes to the API surface, I was just talking about the docs.